README.md

```diff
@@ -53,7 +61,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+weather === "sunny" is a strict assignment.  weather = "sunny" is more loose.
```

Comment:
No, sorry. 
`weather = "sunny"` is the assignment - we're assigning `weather` the value "sunny".
`weather === "sunny"` is a strict _comparison_ - it will compare the two values on either side and evaluate to either `true` or `false` depending on whether or not the two values are **exactly** equal (not 'sorta' equal, which is what the loose equality test, `==`, will check for).